### DISCUSSION TITLE
Increase microphone volume / gain?

### DISCUSSION THREAD

**Discussion by ronsm** [HumanSupportRobot/QandA#26] [Q&A] [answered]
We are having a similar problem to the one discussed in: https://qa.hsr.io/en/question/1633/boosting-microphone-gain/

We we can list and open devices with PyAudio, but the microphone volume seems very low, I have to speak right into it to get anything reasonable. 

I tried to adjust the volume/gain, but we actually cannot see any attached input/output devices in the system settings, and pavucontrol can't connect to the server.

Does anyone have any experience with this? Or with getting decent audio from any of the HSR microphones at all?

Thanks!

**Comment by hsr-admin** [comment, marked as answer]
Please note that I have not verified the operation, so please use this as a reference only.


Set a larger number than 65536, which is 100%.
`$ pacmd set-source-volume alsa_input.usb-OmniVision_Technologies__Inc._USB_Camera-B4.09.24.1-01.multichannel-input 65536`

You can confirm the change with the following command
`$ pacmd list-sources`

**Reply by ronsm** [reply]
Thanks for your reply, this improved things somewhat but the volume is still not great, we are switching to using an external microphone for now.
